src/Promitor.Scraper.Tests.Unit/Serialization/v1/V1IntegrationTests.cs

```diff
@@ -0,0 +1,229 @@
+                        ResourceType = ResourceType.Generic,
+                        Labels = new Dictionary<string, string>
+                        {
+                            {"app", "promitor"}
+                        },
+                        AzureMetricConfiguration = new AzureMetricConfigurationV1
+                        {
+                            MetricName = "ActiveMessages",
+                            Aggregation = new MetricAggregationV1
+                            {
+                                Type = AggregationType.Average
+                            }
+                        },
+                        Resources = new List<AzureResourceDefinitionV1>
+                        {
+                            new GenericResourceV1
+                            {
+                                ResourceUri = "Microsoft.ServiceBus/namespaces/promitor-messaging",
+                                Filter = "EntityName eq 'orders'"
+                            },
+                            new GenericResourceV1
+                            {
+                                ResourceUri = "Microsoft.ServiceBus/namespaces/promitor-messaging",
+                                Filter = "EntityName eq 'accounts'"
+                            }
+                        }
+                    },
+                    new MetricDefinitionV1
+                    {
+                        Name = "promitor_demo_servicebusqueue_queue_size",
+                        Description = "Amount of active messages of the 'orders' queue (determined with ServiceBusQueue provider)",
+                        ResourceType = ResourceType.ServiceBusQueue,
+                        AzureMetricConfiguration = new AzureMetricConfigurationV1
+                        {
+                            MetricName = "ActiveMessages",
+                            Aggregation = new MetricAggregationV1
+                            {
+                                Type = AggregationType.Average,
+                                Interval = TimeSpan.FromMinutes(15)
+                            }
+                        },
+                        Scraping = new ScrapingV1
+                        {
+                            Schedule = "5 4 3 2 1"
+                        },
+                        Resources = new List<AzureResourceDefinitionV1>
+                        {
+                            new ServiceBusQueueResourceV1
+                            {
+                                Namespace = "promitor-messaging",
+                                QueueName = "orders",
+                                ResourceGroupName = "promitor-demo-group"
+                            }
+                        }
+                    }
+                }
+            };
+        }
+
+        [Fact]
+        public void CanDeserializeSerializedModel()
```

Comment:
Can you align the new tests with our current test naming convention please?

Reply:
np

Reply:
I've sorted these - let me know if you spot any more I've missed. It's just an oversight.